src/MaterialDesign3.Demo.Wpf/Buttons.xaml.cs

```diff
@@ -1,5 +1,5 @@
 using System.Diagnostics;
-using MaterialDesign3Demo.Domain;
+using MaterialDesignDemo.Shared.Domain;
 
```

Comment:
using System.Diagnostics;
using MaterialDesign3Demo.Domain;
using MaterialDesignDemo.Shared.Domain;

namespace MaterialDesign3Demo;

Reply:
The `ButtonsViewModel`  used by the `DataContext` has moved from `MaterialDesign3Demo.Domain` to `MaterialDesignDemo.Shared.Domain` which is why I removed line 2 above.

Is there a reason to keep `MaterialDesign3Demo.Domain` even when it is unused?